### PR TITLE
v4.1 upgrade - Farnesene

### PR DESCRIPTION
**Pull request by discoverdefiteam**

### CODE REVIEW

**Comment by discoverdefiteam** [comment]
covers #156

File: app/keepers/keepers.go [hunk 1]
```diff
@@ -92,25 +92,27 @@ import (
...
+	// cwhookstypes "github.com/terpnetwork/terp-core/v4/x/cw-hooks/types"
 
 	"github.com/terpnetwork/terp-core/v4/x/tokenfactory/bindings"
 	tokenfactorykeeper "github.com/terpnetwork/terp-core/v4/x/tokenfactory/keeper"
 	tokenfactorytypes "github.com/terpnetwork/terp-core/v4/x/tokenfactory/types"
 )
 
 var (
-	wasmCapabilities = "iterator,staking,stargate,token_factory,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4,tokenfactory"
+	wasmCapabilities = "iterator,staking,stargate,token_factory,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4,cosmwasm_1_5,tokenfactory"
```
Comment (Reecepbcups) [inline app/keepers/keepers.go:115]: There are 2 token_factory and tokenfactory

File: app/upgrades/v4_1/constants.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+// UpgradeName defines the on-chain upgrade name for the Terp v4 upgrade.
+const UpgradeName = "v4.1"
+
+var Upgrade = upgrades.Upgrade{
+	UpgradeName:          UpgradeName,
+	CreateUpgradeHandler: CreateV4_1UpgradeHandler,
+	StoreUpgrades: store.StoreUpgrades{
+		Added: []string{
+			driptypes.ModuleName,
```
Comment (Reecepbcups) [inline app/upgrades/v4_1/constants.go:18]: did v4 get into prod with the burn module? if so make sure to Remove that store key here, not just remove the keeper

Comment (hard-nett) [reply app/upgrades/v4_1/constants.go:18]: Appreciate you following up here! The burn module only made it to our test network `90u-2`. Is it certain the burn module needs the store key removed? We do not have a store key specified [here](https://github.com/terpnetwork/terp-core/blob/v4.0.0/x/burn/burner.go), and we also did not specify the module stored in the [v4 upgrade](https://github.com/terpnetwork/terp-core/blob/v4.0.0).

Comment (Reecepbcups) [reply app/upgrades/v4_1/constants.go:18]: ahh good call. was a busy day and forgot it's more of a plugin vs module

so my comment does no apply, you're all good

Comment (hard-nett) [reply app/upgrades/v4_1/constants.go:18]: no worries, appreciate you!

**Pull request merged**